### PR TITLE
add default performance profile

### PR DESCRIPTION
**Pull request by khansaad**
This PR contains following changes:

- Add default performance profile and corresponding class
-  update interface data and related files

Signed-off-by: saakhan <saakhan@redhat.com>

### CODE REVIEW

File: src/main/java/com/autotune/analyzer/utils/ExperimentResultValidation.java [hunk 1]
```diff
@@ -55,18 +63,56 @@ public void validate(List<ExperimentResultData> experimentResultDataList) {
...
+                            Class<?> validationClass = null;
+                            Class<?>[] parameterTypes = null;
+                            Method method = null;
+                            Object object = null;
+                            // check if performance profile is absent and then attach a default one
+                            if (null == performanceProfile) {
+                                SloInfo sloInfo = kruizeObject.getSloInfo();
+                                performanceProfile = new PerformanceProfile(AnalyzerConstants.PerformanceProfileConstants.DEFAULT_PROFILE,AnalyzerConstants.DEFAULT_PROFILE_VERSION, AnalyzerConstants.DEFAULT_K8S_TYPE, sloInfo);
+                                kruizeObject.setPerformanceProfile(performanceProfile.getName());
+                                errorMsg = new DefaultOpenshiftImpl().validate(performanceProfile, resultData);
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/utils/ExperimentResultValidation.java:81]: Why is this being called `DefaultOpenshiftImpl`? This has nothing to do with OpenShift. It is a generic profile for all experiments that don't define a profile

Comment (khansaad) [reply src/main/java/com/autotune/analyzer/utils/ExperimentResultValidation.java]: Updated

**Comment by dinogun** [comment]
@khansaad we have some conflicts as I merged Bharath's PR #565

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**